CHANGELOG.next.toml

```diff
@@ -159,16 +159,16 @@ meta = { "breaking" = true, "tada" = false, "bug" = false }
 author = "ysaito1001"
 
 [[aws-sdk-rust]]
-message = "Update MSRV to Rust 1.68.2"
-references = ["smithy-rs#2745"]
+message = "Update MSRV to Rust 1.69.0"
+references = ["smithy-rs#2893"]
 meta = { "breaking" = true, "tada" = false, "bug" = false }
-author = "jdisanti"
+author = "Velfi"
```

Comment:
For the sake of keeping records, do we want to leave John's change log entries and then add new change log entries for updating to 1.69.0 (i.e. keep both `A->B` and `B->C`, instead of `A->C`)?